platforms/models.py

```diff
@@ -24,3 +24,11 @@ class Word(models.Model):
 
     def __str__(self):
         return self.word
+
+class UsersWords(models.Model):
```

Comment:
class name for models: noun over plural 
```suggestion
class UserWord(models.Model):
```